recipes-samples/images/rpb-console-image-lava.bb

```diff
@@ -1,5 +1,7 @@
 require rpb-console-image.bb
 
+IMAGE_ROOTFS_EXTRA_SPACE = "1048576"
```

Comment:
our images  include 96boards-tools, which installs resize-helper.service , which is supposed to resize the file system to the partition size on first boot. so if that was working fine, we wouldn't need this patch. 

i remember the resize-helper script was broken on master for some time, we fixed it here https://github.com/96boards/96boards-tools/commits/master, but we need to update the recipe.. instead of this patch, we need to release 0.11 version of 96boards-tools and upgrade the recipe.

Reply:
We never updated the recipe when 96boards-tools has been broken so we don't really need to update to latest version. Upgrading to latest release is orthogonal.

Reply:
i think we did. 0.10 includes https://github.com/96boards/96boards-tools/commit/906d4bde05b1e2d276deebecd789b53675fd7922, which is the commit that broke the automatic resize at boot

Reply:
Ok, i didn't know we have such tool.